lib/rubocop/node_pattern.rb

```diff
@@ -429,6 +429,7 @@ def compile_ellipsis
         [0..Float::INFINITY, 'true']
       end
 
+      # rubocop:disable Metrics/AbcSize
```

Comment:
Couldn't find any way to refactor this method that didn't make it worse. Pretty sure this is as good as it gets with the current design, but open to suggestions.

Reply:
Challenge accepted 😉 https://github.com/rubocop-hq/rubocop/pull/7457